### DISCUSSION TITLE
Discord checkbox for problems

### DISCUSSION THREAD

**Discussion by Tealk** [pelican-dev/panel#526] [General]
![image](https://github.com/user-attachments/assets/76d10309-f741-497f-b45a-9f92d9fc6a17)

The last checkbox I find a little difficult, I have no discord and I do not want to use this spyware, but without this checkbox you can not open an issue.

**Comment by QuintenQVD0** [comment]
> ![image](https://private-user-images.githubusercontent.com/12276250/355041032-76d10309-f741-497f-b45a-9f92d9fc6a17.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NTE3MjIsIm5iZiI6MTcyMjg1MTQyMiwicGF0aCI6Ii8xMjI3NjI1MC8zNTUwNDEwMzItNzZkMTAzMDktZjc0MS00OTdmLWI0NWEtOWY5MmQ5ZmM2YTE3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA1VDA5NTAyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxNGIzYjMzYjU0ODQzYTExOTcyNDZhNTc5ZmExZmQ5MTg2NjRjM2Q4Zjc4MGJjODFjYjg3OWIyZjQwNjFhMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yHmZOd6SKFM72FKEADYHsZGmMMrfhiHVKkz_yp4CEMA)
> 
> The last checkbox I find a little difficult, I have no discord and I do not want to use this spyware, but without this checkbox you can not open an issue.

What makes you think discord is spyware?

**Reply by Tealk** [reply]
Here is a nice summary: https://spyware.neocities.org/articles/discord

**Comment by Boy132** [comment]
We provide support only via Discord. If you don't want to use discord that's on you.
Also, discord is not a "spyware".

**Reply by Tealk** [reply]
But that means I can't create issues without using discord.

**Reply by QuintenQVD0** [reply]
Ngl. It always have been this way and it wont be changing anytime soon.

**Comment by Sharktheone** [comment]
Must be hell for you to have an Github Account... I mean, Github is closed source, owned by Microsoft. You probably couldn't play a single game with that mentality. The website you liked rated Unity and Steam as "Extremely High" spyware, Duckduckgo is also "Possible Spyware", the Tor browser's spyware level is "Low".

I also think that calling basically everything "spyware" makes actual spyware seem less harmful than it is.
